github-actions/rebase-pr/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.10
+
+RUN apk --no-cache add jq bash git
+
+ADD entrypoint.sh /entrypoint.sh
```

Comment:
nit: COPY is safer than ADD (COPY only supports files; ADD can access remote files via a URL)